src/main/java/io/github/bobocodebreskul/context/annotations/BringBean.java

```diff
@@ -0,0 +1,12 @@
+package io.github.bobocodebreskul.context.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface BringBean {
```

Comment:
No documentation. 
Good to have usage example.

Reply:
added